project/metals.sbt

```diff
@@ -0,0 +1,4 @@
+// DO NOT EDIT! This file is auto-generated.
+// This file enables sbt-bloop to create bloop config files.
+
+addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.4.8")
```

Comment:
This file should probably be added to the .gitignore. (same for the other two `metals.sbt` files).

Reply:
Oops, sorry.